packages/plugin-core/src/test/suite-integ/migration.test.ts

```diff
@@ -252,5 +252,72 @@ suite("Migration", function () {
         },
       });
     });
+
```

Comment:
can we add a test for migrating keyboard shortcuts? reminder to mock the keyboard shortcuts dir/file (like how we currently mock the home directory) so the tests don't override our actual shortcuts